unfetter-discover/unfetter-ui
# bar chart for threat dashboard

bar chart for threat dashboard, fixed closing toolbox, fixed typedoc generation

fixes unfetter-discover/unfetter#638

## To Test
* pull this branch
* npm run doc, npm run test, npm run build:prod, npm run lint; verify no errors
* visit, the threat dashboard page, verify a bar chart and that the ratios match the radar and table views
